### PR TITLE
[IMP] Recruitment: updating images and instructions for 17

### PR DESCRIPTION
**Pull request by larm-odoo**
Need to update the doc to reflect the changes made in 17

### CODE REVIEW

**Comment by larm-odoo** [comment]
This one is ready for review, @jero-odoo.

**Comment by larm-odoo** [comment]
Hey @hojo-odoo, I don't think I've had a review form you in a long time, so I am tagging you for my peer review. Thank you!

**Comment by StraubCreative** [comment]
Fresh rebase on [89edc58](https://github.com/odoo/documentation/pull/7869/commits/89edc58d0e891f559bdda0e159439f183f826e1b) to rebuild the runbot instance 👍

File: content/applications/hr/recruitment.rst
Comment (StraubCreative) [inline content/applications/hr/recruitment.rst:72]: Suggestion to similarly acknowledge the internal link that shows up when the feature is enabled, and to break off the last sentence from the paragraph.

```suggestion
enable the :guilabel:`Send SMS` option. This option requires credits to use, which can be obtained
by clicking the :guilabel:`➡️ Buy credits` internal link that emerges when the feature is enabled.

For more information,
```

File: content/applications/hr/recruitment.rst
Comment (StraubCreative) [inline content/applications/hr/recruitment.rst:81]: think this is redundant?
```suggestion
When applicants submit an application, one of the required fields is a resumé, or CV. All
```

Comment (StraubCreative) [reply content/applications/hr/recruitment.rst:81]: Also might be neat to add an **:abbr:** tag here to define **CV (curriculum vitae)** 😄

File: content/applications/hr/recruitment.rst
Comment (StraubCreative) [inline content/applications/hr/recruitment.rst:85]: Specificity question around location: 
Is the CV available on the _card_ or to the right of the _form_, once the card is clicked into?

File: content/applications/hr/recruitment.rst
Comment (StraubCreative) [inline content/applications/hr/recruitment.rst:91]: Consider formatting this as a **note**, perhaps since it's supplemental information having to do with browser width.

File: content/applications/hr/recruitment.rst
Comment (StraubCreative) [inline content/applications/hr/recruitment.rst:103]: Extract *where*, exactly?
Into the applicant card? Contact record?

File: content/applications/hr/recruitment.rst
Comment (StraubCreative) [inline content/applications/hr/recruitment.rst:111]: Curious: is it possible to digitize in bulk, maybe in a list view group selection?

**Review by StraubCreative — COMMENTED** [review]
Hi @larm-odoo 

Here's my first round of peer review, to get you started! 

As always, your documentation is so detailed and really impressive 🙂 Please see my comments for you below for you to consider, as they mostly have to do with the content itself and perhaps expanding UI coverage in some spots. You'll notice a few writing-specific comments as well, where I caught them (however I leave the majority of those suggestions up to Final Review 😉).

Please let me know if you have any questions on this batch.
I'll add more comments soon!

File: content/applications/hr/recruitment.rst
Comment (larm-odoo) [reply content/applications/hr/recruitment.rst:61]: I love this idea! I linked ot the surveys essentials for now, and then when that doc is created it can be referenced instead.

File: content/applications/hr/recruitment.rst
Comment (larm-odoo) [reply content/applications/hr/recruitment.rst:85]: It's the form (I thought if it as the selected card). It's fully explained now.

File: content/applications/hr/recruitment.rst
Comment (larm-odoo) [reply content/applications/hr/recruitment.rst:111]: I don't believe so- I tried myself by clicking and draggin, or importing, or creating new and adding several- looks lik eyu can only do one at a time. And online form the application form, only one. The ONYL thing I think could happen is if you emailed the email alias several applications at once, but I don't think that works- I am checking with the experts if there is a way to do that. Although between you, me, and the wall, since it imports such litte informatio (the bare mnimum) it almostdoesn't seem worth it, if the oly way is to email. But I'll update when I hear back! In the meantime, I expanded this a little.

**Review by StraubCreative — APPROVED** [review]
Hi @larm-odoo 
Thorough as always!
I just had some high-level stuff for you to consider here. Feel free to apply what you agree with and then you can tag SALI or me again for a tech review (whoever can get to it first).
Thanks for the great work here!

File: content/applications/hr/recruitment.rst
Comment (larm-odoo) [reply content/applications/hr/recruitment.rst:64]: True- this was just suc an old PR, lol. I'll be sure to do all of those in the new format now!

**Comment by larm-odoo** [comment]
This is ready for a tech review when you have a chance, @samueljlieber! If you are busy, @StraubCreative can do it. Whoever gets to it first, really. =)

File: content/applications/hr/recruitment.rst
Comment (samueljlieber) [inline content/applications/hr/recruitment.rst:593]: There is no text on the UI stating "Quick add", so you can leave it lowercase and state it is a button, whereas the icon is the + sign
```suggestion
ways, using either the :ref:`quick add <recruitment/quick-add-applicant>` button or the :ref:`New
<recruitment/create-new-applicant>` button.
```

File: content/applications/hr/recruitment.rst
Comment (samueljlieber) [inline content/applications/hr/recruitment.rst:600]:
```suggestion
Quickly add a new applicant using the *quick add* button. If not already in the job position Kanban
```

File: content/applications/hr/recruitment.rst
Comment (samueljlieber) [inline content/applications/hr/recruitment.rst:604]:
```suggestion
view, navigate to the main recruitment dashboard by going to :menuselection:`Recruitment app -->
Applications --> By Job Positions`. Click the :guilabel:`(#) New Applications` button on the job
position card that the applicant should be added to. Then, click on the small :icon:`fa-plus`
:guilabel:`(plus)` icon in the top-right of the :guilabel:`New` stage to quickly add a new
applicant.
```

File: content/applications/hr/recruitment.rst
Comment (samueljlieber) [inline content/applications/hr/recruitment.rst:641]:
```suggestion
Add a new applicant including all the relevant information using the *New* button. If not already in
the Kanban view for the job position to add an applicant to, navigate to the main recruitment
dashboard by going to :menuselection:`Recruitment app --> Applications --> By Job Positions`. Then,
click the :guilabel:`New` button in the top-left of the Kanban view and a blank applicant form
loads.
```

File: content/applications/hr/recruitment.rst
Comment (samueljlieber) [inline content/applications/hr/recruitment.rst:1052]: Use guilabel for text displayed on the UI
```suggestion
a red box at the bottom of the :guilabel:`Generate a Simulation Link` pop-up window, stating
:guilabel:`The applicant does not have a valid email set. The Offer Link won't be able to be
completed.` Click :guilabel:`Discard`, enter an email on the applicant's card, then click the
:guilabel:`Generate Offer` button again.
```

**Review by samueljlieber — APPROVED** [review]
Hi @larm-odoo! Thanks for your thorough improvements to the Recruitment doc! I am approving with some comments, please let me know if you have any questions! Thank you for your hard work 😊
...
@robodoo delegate=larm-odoo

File: content/applications/hr/recruitment.rst
Comment (StraubCreative) [reply content/applications/hr/recruitment.rst:49]: Good catch Sam.
Alternatively a descriptive noun would complete the sentence, after the gui

```suggestion
   The :guilabel:`Online Posting` feature only works if the *Website* application is also installed.
```

File: content/applications/hr/recruitment.rst
Comment (larm-odoo) [reply content/applications/hr/recruitment.rst:146]: I missed those little guys!

**Comment by larm-odoo** [comment]
Than you @samueljlieber! I really liked the tbale- I didn't even think of adding a table for that section =). Everythign haas been updated- and once the checks pass, it's be merged!

**Comment by larm-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
